src/androidTest/java/de/blau/android/dialogs/BookmarkDialogsTest.java

```diff
@@ -0,0 +1,116 @@
+package de.blau.android.dialogs;
+
+
+import android.app.Instrumentation;
+import android.app.Instrumentation.ActivityMonitor;
+
+import androidx.test.uiautomator.UiDevice;
+import androidx.test.uiautomator.UiObject;
+import androidx.test.uiautomator.UiObjectNotFoundException;
+import androidx.test.uiautomator.UiSelector;
+import android.content.Intent;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import androidx.test.platform.app.InstrumentationRegistry;
+import androidx.test.rule.ActivityTestRule;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import java.util.ArrayList;
+
+
+import de.blau.android.Main;
+import de.blau.android.Map;
+
+import de.blau.android.Splash;
+import de.blau.android.TestUtils;
+import de.blau.android.bookmarks.BookmarksStorage;
+import de.blau.android.exception.OsmException;
+import de.blau.android.osm.ViewBox;
+
+import static org.junit.Assert.assertNotNull;
+
+@RunWith(AndroidJUnit4.class)
+public class BookmarkDialogsTest {
+
+    Instrumentation instrumentation = null;
+    ArrayList<BookmarksStorage> bookmarksStorages = null;
+    ViewBox     viewBoxtest = null;
+    ActivityMonitor monitor = null;
+    UiDevice        device  = null;
+    Splash          splash  = null;
+    Main            main    = null;
+    Map             map     = null;
+
+    @Rule
+    public ActivityTestRule<Splash> mActivityRule = new ActivityTestRule<>(Splash.class, false, false);
```

Comment:
The issue with starting with the Splash activity is that it installs the background imagery configuration, this naturally looks nice when the test is being run, because you can visually see if it is working, but implies that when the test runs we are trying to access external services.

In any case I would directly start Main here to avoid this issue.